browser/js/common/factories/ItemFactory.js

```diff
@@ -18,6 +18,24 @@ app.factory('ItemFactory', function($http) {
 			return res.data;
 		});
 	};
+
+	factory.getUser = function (productId, userId){
+		return $http.get('/api/item/' + productId + '/user/' + userId).then(function (res){
+			return res.data.sellerID;
+		});
+	};
+	factory.getReviews = function (productId){
+		return $http.get('/api/item/' + productId + '/reviews').then(function (res){
+			return res.data.reviews;
+		});
+	};
+	factory.getReviewUser = function (userId){
```

Comment:
I would update the route so that this is more clear. From the route, it's hard to tell that it's different from getUser().